core/designsystem/src/main/kotlin/com/google/samples/apps/nowinandroid/core/designsystem/component/scrollbar/ScrollbarExt.kt

```diff
@@ -102,3 +104,40 @@ fun LazyGridState.scrollbarState(
         },
         reverseLayout = { layoutInfo.reverseLayout },
     )
+
+/**
+ * Remembers a [ScrollbarState] driven by the changes in a [LazyStaggeredGridState]
+ *
+ * @param itemsAvailable the total amount of items available to scroll in the staggered grid.
+ * @param itemIndex a lookup function for index of an item in the staggered grid relative
+ * to [itemsAvailable].
+ */
+@Composable
```

Comment:
Is `:core::designsystem` correct module? Wouldn't `:core:ui` be more appropriate?

Reply:
Good point, I went with `core:designsystem` bc all the other custom NiA components like progress bars and buttons were here. I can leave the NiA specific scrollbar style here and move the internal workings to core:ui?

Reply:
It seems `:core:ui` depends on `:core:designsystem`, so I can't move it. I'm not sure why the dependency setup is this way, but it seems out of scope for this PR.